app/src/main/java/com/example/demomvvm/data/repository/source/remote/api/response/BaseResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.demomvvm.data.repository.source.remote.api.response
+
+import com.google.gson.annotations.SerializedName
+
+class BaseResponse<T> (
+
+    @SerializedName("results")
```

Comment:
cái này là constructor mà thanh niên
```suggestion
class BaseResponse<T> (
    @SerializedName("results")
```